Automattic/jetpack
# Subscribe modal: update template markup and adjust styles for easier modification and mobile compatibility

## Proposed changes:

### Simplifies template blocks

Removes one row block to simplify block structure. Thus making it easier to edit for customers. 


### All the styles within template and possible to customize

Remove all external styling affecting the modal's contents, leaving everything for the blocks and theme styles. This will allow customers entirely re-design the modal. E.g. previously background color change wouldn't affect entire modal.

**BEFORE**

<img width="710" alt="Screenshot 2023-08-02 at 17 11 53" src="https://github.com/Automattic/jetpack/assets/87168/735e6fca-dce0-44ac-9990-60652ae19444">


**AFTER**

<img width="759" alt="Screenshot 2023-08-02 at 16 18 27" src="https://github.com/Automattic/jetpack/assets/87168/8c12ac83-b376-4992-b5ee-69813edcae17">
<img width="696" alt="image" src="https://github.com/Automattic/jetpack/assets/87168/aee81cf1-6fc7-43b2-b38d-23c35cd63829">


### Fix mobile styles
Removes `width: 600px` to fix broken mobile styles.

**BEFORE**

<img width="423" alt="Screenshot 2023-08-02 at 17 12 10" src="https://github.com/Automattic/jetpack/assets/87168/d9a360f9-50c6-4710-ba04-215f2c2db2c7">

**AFTER**

<img width="427" alt="Screenshot 2023-08-02 at 16 17 46" src="https://github.com/Automattic/jetpack/assets/87168/fa1bf2d5-7a94-4fe8-a6ee-c95d5f1e7a0b">


### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






- Force neable modal with `add_filter( 'jetpack_subscriptions_modal_enabled', '__return_true', 20 );`
- Enable modal from jetpack settings
- In the frontend, make sure you don't have the cookie stored which disables the modal for a day
- Open any blog post, scrolla bit down
- See the modal, test different browser widths
